example-get-started-experiments/generate.sh

```diff
@@ -117,8 +117,6 @@ dvc stage add -n evaluate \
   -d src/evaluate.py -d models/model.pkl -d data/test_data \
   python src/evaluate.py
 
-rm results/train/dvc.yaml
-git rm --cached results/train/dvc.yaml
```

Comment:
What's this change about?

Reply:
It was unnecessary. I don't recall why it was there. I assume that in an older version, we used to define the metrics and plots in the top level dvc.yaml so this was introduced to prevent duplication